webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js

```diff
@@ -109,6 +109,7 @@ const ActionsBar = ({
       key="console"
       isAriaDisabled={isConsoleDisabled}
       tooltip={determineTooltip()}
+      isDisabled={!computeId || !isHostActive}
```

Comment:
Is this change related or this is just a side noted fix? If later, then it should probably be extracted in case we would want to CP it.

Reply:
That change is not related, you are right.